test/actions/BackendAction-test.js

```diff
@@ -265,6 +271,31 @@ describe('BackendAction', () => {
       }
     })
 
+    it('should not write generated extension-config if backend-extension was updated but extension is on blacklist', async () => {
+      let generated = { id: 'myIgnoredExtension', backend: { id: 'myIgnoredExtension' } }
+
+      let called = 0
+      subjectUnderTest.dcHttpClient.generateExtensionConfig = () => {
+        called++
+        return Promise.resolve(generated)
```

Comment:
return Promise.resolve() would be enough here I guess. Or nothing at all. It's not even expected to be called anyway, right? You could also place an assert.fail() into the body, not sure if counting is necessary with only one extension tested?

Reply:
done